black8/assets/.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
@taion how do I make sure this folder is packaged in the pypi package?

Reply:
set `package_data` in `setup()` in `setup.py`... see what we do in our internal tools repo